tests/python/test_stop_grad.py

```diff
@@ -70,9 +70,9 @@ def func():
         # Two loops, one with stop grad on without
         for i in range(n):
             ti.stop_grad(x)
-            ti.atomic_add(loss, x[i]**2)
```

Comment:
Why no `[None]`? Does this work or we needs this?

Reply:
That works but is not the standard syntax.